### PR TITLE
Rough draft of owner aliases

### PR DESCRIPTION
**Pull request by asberardinelli**
- Sets manager (owner( aliases pretty far upstream so that Owners and Matchups use them properly
- Consolidate manager name cleaning into a utility function
- Need to get actual aliases from managers
- Need to update Meet the Manager bio keys with properly formatted keys from new aliases

Making draft PR for now so y'all can take a look, will want to address the last two bullet points before merging

### CODE REVIEW

File: fantasy_matchup.py [hunk 1]
```diff
@@ -37,10 +37,10 @@ def __init__(self, owner, team, year, week, matchup):
 		self.team_name = team
 		self.week = week
 		self.year = year
-		if owner == matchup.home_team.owner.replace("  ", " ").strip().title():
+		if owner == matchup.home_team.owner:
```
Comment (kmanc) [inline fantasy_matchup.py:40]: Do these need to have clean_name run on them?

File: fantasy_matchup.py [hunk 2]
```diff
@@ -37,10 +37,10 @@ def __init__(self, owner, team, year, week, matchup):
...
 			self.score = matchup.home_score
 			try:
-				self.opponent_owner_name = matchup.away_team.owner.replace("  ", " ").strip().title()
+				self.opponent_owner_name = matchup.away_team.owner
```
Comment (kmanc) [inline fantasy_matchup.py:43]: Above

File: fantasy_matchup.py [hunk 3]
```diff
@@ -54,7 +54,7 @@ def __init__(self, owner, team, year, week, matchup):
 		else:
 			self.score = matchup.away_score
 			try:
-				self.opponent_owner_name = matchup.home_team.owner.replace("  ", " ").strip().title()
+				self.opponent_owner_name = matchup.home_team.owner
```
Comment (kmanc) [inline fantasy_matchup.py:57]: Above

File: utility.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+MANAGER_ALIASES = {
+	"Joe Guidoboni" : "Poopoo Peepants",
+	"Brendan Shea" : "Real League Daddy",
+}
+
+def clean_name(name: str) -> str:
+    """Cleans up a manager's name str and fetches its alias, if present"""
+    cleaned = name.replace("  ", " ").strip().title()
+    return MANAGER_ALIASES[cleaned] if cleaned in MANAGER_ALIASES else cleaned
```
Comment (kmanc) [inline utility.py:9]: I think you can simplify this last line with a little bit of Python dict fanciness

```
return MANAGER_ALIASES.get(cleaned, cleaned)
```

**Review by kmanc — CHANGES_REQUESTED** [review]
Dropped a few comments, but overall super excited for this!

File: utility.py (hunk 4 above)
Comment (asberardinelli) [reply utility.py:9]: good call, much cleaner. updating

File: fantasy_matchup.py (hunk 1 above)
Comment (asberardinelli) [reply fantasy_matchup.py:40]: Yeah I'm actually hitting an issue I missed early related to these lines, kinda related anyway, I have to make an update to here

Comment (asberardinelli) [reply fantasy_matchup.py:40]: Fixed a change I made further upstream (`fantasy_league.py` lines ~115-125) that was intended to make it so we don't need `clean_name` here.  So now we actually don't need it, nor do we need the `try/except`s here anymore

Comment (kmanc) [reply fantasy_matchup.py:40]: Oh hell yeah!! Nice work!

**Comment by kmanc** [comment]
Haven't paid too close attention to League members' responses, but are we good to update the names and get the site back up?

**Comment by BryanRockwood** [comment]
If we push the name change (just first names) I think we are good to go - I
guess we should also check the blurbs

**Comment by kmanc** [comment]
Made a few small changes, testing locally appeared to work fine (if your espn_api version is less than 0.30.0 it might not work). @asberardinelli I think all we need now is the bios and we're good to go

**Review by kmanc — DISMISSED** [review]

**Review by kmanc — APPROVED** [review]

**Pull request merged**